assets/src/pages/room/components/StreamPlayer/simulcast/SimulcastEncodingToReceive.tsx

```diff
@@ -1,30 +1,80 @@
 import React, { FC } from "react";
 import { TrackEncoding } from "@jellyfish-dev/membrane-webrtc-js";
-import { isTrackEncoding } from "../../../../types";
+import { Tooltip } from "./Tooltip";
+import { LayerButton } from "./LayerButton";
 
 type Props = {
-  setDesiredEncoding: (encoding: TrackEncoding) => void;
-  desiredEncoding?: TrackEncoding;
-  disabled?: boolean;
+  targetEncoding: TrackEncoding | null;
+  setTargetEncoding: (encoding: TrackEncoding) => void;
+  tileSizeEncoding: TrackEncoding | null;
+  smartEncoding: TrackEncoding | null;
+  localSmartEncodingStatus: boolean;
+  setLocalSmartEncodingStatus: (value: boolean) => void;
+  globalSmartEncodingStatus: boolean;
+  disabled: boolean;
 };
 
-export const SimulcastEncodingToReceive: FC<Props> = ({ desiredEncoding, setDesiredEncoding, disabled }: Props) => {
+export const SimulcastEncodingToReceive: FC<Props> = ({
+  targetEncoding,
+  setTargetEncoding,
+  tileSizeEncoding,
+  smartEncoding,
+  localSmartEncodingStatus,
+  setLocalSmartEncodingStatus,
+  globalSmartEncodingStatus,
+  disabled,
+}: Props) => {
   return (
-    <div className="absolute bottom-0 right-0 z-50 bg-white p-2 text-sm text-gray-700 opacity-80 md:text-base">
-      <label>Encoding to receive</label>
-      <select
-        disabled={disabled}
-        value={desiredEncoding}
-        onChange={(event) => {
-          const value = event.target.value;
-          if (!isTrackEncoding(value)) return;
-          setDesiredEncoding(value);
-        }}
-      >
-        <option value="h">High</option>
-        <option value="m">Medium</option>
-        <option value="l">Low</option>
-      </select>
+    <div className="absolute bottom-0 right-0 z-50 w-full bg-white p-2 text-sm text-gray-700 opacity-80 md:text-base">
+      <div className="flex flex-row justify-between">
+        <Tooltip text="Toggle automatic layer switching" textCss="left-24">
+          <div className="form-check flex items-center gap-x-1">
+            <label className="form-check-label text-brand-dark-blue-500">Smart</label>
+            <input
+              disabled={!globalSmartEncodingStatus}
+              onChange={() => setLocalSmartEncodingStatus(!localSmartEncodingStatus)}
+              className="form-check-input"
+              type="checkbox"
+              checked={globalSmartEncodingStatus && localSmartEncodingStatus}
+            />
+          </div>
+        </Tooltip>
+
+        <Tooltip text="Encoding based on all rules">
+          <div>Smart: {smartEncoding}</div>
+        </Tooltip>
+
+        <Tooltip text="Encoding based on tile size" textCss="right-20">
+          <div>Tile: {tileSizeEncoding}</div>
+        </Tooltip>
+      </div>
+
+      <div className="flex flex-row justify-between">
+        <Tooltip text="Selected encoding target" textCss="left-20">
+          <div>Encoding: {targetEncoding}</div>
```

Comment:
Due to the size and placement of this tooltip, it is almost impossible to switch the `smart` option, since the tooltip is overlapping it.
<img width="673" alt="Screenshot 2023-02-21 at 13 18 15" src="https://user-images.githubusercontent.com/58188916/220343655-f6e78944-5d7a-400c-a23a-6c96cd627260.png">

Reply:
You're right. But these options are only for developers so I decided not to waste time finding a better solution. I could swap left and center items in this row because center one is not interactive